centos-ci/bootstrap.sh

```diff
@@ -51,5 +51,5 @@ export DISPLAY=:19
 
 
 echo "$(print_date): running integration tests"
-time scl enable rh-python35 sclo-vagrant1 -- sh -xc "cd ../.. && pipenv shell 'cd integration-tests && behave --no-color --junit; exit \\\$?'"
+time scl enable rh-python35 sclo-vagrant1 -- sh -xc 'cd ../../integration-tests && behave --no-color --junit; exit \\\$?'
```

Comment:
Aye, if it's SCL related, that would explain why I couldn't reproduce it locally. I've tried to force another failure based on your experiment above (skipping the full test run this time), so if that fails as expected, I'll be able to remove the forced failure and this will be ready to go.